lib/run-script.js

```diff
@@ -7,7 +7,7 @@ var lifecycle = require("./utils/lifecycle.js")
   , log = require("npmlog")
   , chain = require("slide").chain
 
-runScript.usage = "npm run-script <command> [-- <args>]"
+runScript.usage = "npm run-script <command>|env [-- <args>]"
```

Comment:
I don't like special-casing this in the usage statement. We don't do the same for `npm run-script test`. Leave the complete documentation to the manpage.